packages/aws/data_stream/ec2_metrics/_dev/deploy/tf/main.tf

```
@@ -10,6 +10,11 @@ provider "aws" {
       branch       = var.BRANCH
       build        = var.BUILD_ID
       created_date = var.CREATED_DATE
+
+      division = "engineering"
+      org      = "ingest"
+      team     = "ecosystem"
+      project  = "ci-testing"
```

We should probably add here the owner of the package. And maybe add a project related to this package.
```suggestion
      org      = "observability"
      team     = "obs-cloud-monitoring"
      project  = "integrations-aws-package"
```

---

Ok! Organization for observability should be `obs`.
I'll update the other tags as your suggestion